assembly/release-apache-ignite-base.xml

```diff
@@ -36,15 +36,9 @@
         </file>
 
         <file>
-            <source>modules/platforms/cpp/configure.acrel</source>
+            <source>modules/platforms/cpp/CMakeLists.txt</source>
```

Comment:
We didn't copy it before? What was affected?

Reply:
Yes, I forgot this in the first commit.